src/xcode/ENA/ENA/Source/Scenes/Events/Store/Model/__test__/TraceLocationTypeTest.swift

```diff
@@ -0,0 +1,127 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class TraceLocationTypeTest: XCTestCase {
+
+	func testGIVEN_locationTypeUnspecified_THEN_TitleAndSubtitleMatch() {
+		// GIVEN
+		let traceLocationType: TraceLocationType = .locationTypeUnspecified
+
+		// THEN
+		XCTAssertEqual(AppStrings.TraceLocations.unspecified.title, traceLocationType.title)
```

Comment:
he order is a bit unusual here for all the tests, usually we compare the other way around

```suggestion
		XCTAssertEqual(traceLocationType.title, AppStrings.TraceLocations.unspecified.title)
```

Reply:
I did not know we have an order for that, but can change it ;-)